src/pepr/operator/controllers/packages/package-store.ts

```diff
@@ -173,11 +181,43 @@ function findPackagesWithSsoClientId(clientId: string): Set<string> {
   return ssoIndex.get(clientId) ?? new Set<string>();
 }
 
+/**
+ * Finds all packages that have ambient waypoint enabled
+ * @returns Array of UDSPackage objects with ambient waypoint enabled
+ */
+function getAmbientPackages(): UDSPackage[] {
+  const result: UDSPackage[] = [];
+  for (const namespaceMap of packageNamespaceMap.values()) {
+    for (const pkg of namespaceMap.values()) {
+      if (pkg.spec?.network?.serviceMesh?.mode === "ambient") {
```

Comment:
nit: silly q - is there a reason why you can't compare the actual enum like so?
```suggestion
      if (pkg.spec?.network?.serviceMesh?.mode === Mode.Ambient) {
```

Reply:
OBE